src/qmapshack/gis/CGisListWks.cpp

```diff
@@ -1059,6 +1060,7 @@ void CGisListWks::showMenuItemTrk(const QPoint& p, const IGisItem::key_t& key)
     menu.addAction(actionEleWptTrk);
     menu.addAction(actionCopyTrkWithWpt);
     menu.addAction(actionNogoTrk);
+    menu.addAction(actionToRoute);
```

Comment:
Keep the order of the actions in the menu the same as the tool buttons in the on screen boxes. Users will appreciate it :wink: